agent/doc/swdesign/README.md

```diff
@@ -80,17 +80,48 @@ The following diagram shows the startup sequence of the Ankaios Agent:
 
 Status: approved
 
-The Ankaios Agent shall use the given interfaces and channels to communicate with the Server.
+The Ankaios agent shall use the given interfaces and channels to communicate with the server.
 
 Tags:
 - AgentManager
 
-Rationale: The Server is "only source of true" and ensures that Agents are in the consistent state.
+Rationale: The server is "only source of true" and ensures that Agents are in the consistent state.
 
 Needs:
 - impl
 - itest
 
+#### Agent supports environment variable for the server url
+`swdd~agent-shall-support-environment-variable-for-server-url~1`
+
+Status: approved
+
+When the environment variable with key `ANKAIOS_SERVER_URL` is set and the user has not provided the server url via cli argument,
```

Comment:
I would prefer, if we cloud derive from the name of an env var the tools it belongs to, e.g.

- `ANK_...` modifies the behavior of the Ankaios cli
- `ANKAGENT_...` modifies the behavior of the Ankaios agent
- `ANKSERVER_...` modifies the behavior of the Ankaios server

So using that notation the variable should be called `ANKAGENT_SERVER_URL`.

Reply:
Also I am not sure if we need a separate requirement for every environment variable that we introduce. Just an idea ...

Reply:
@krucod3 do you agree with having prefixes per component?